nestor/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""nestor-api main module."""
+
+__author__ = "Kapten"
+__email__ = "tech-admin@kapten.com"
```

Comment:
Maybe we could put `tech@kapten.com` just like in `setup.py` (I am not sure we will be able to keep this address for long, though

Reply:
I'll just set `tech-admin` everywhere.